pinax/referrals/views.py

```diff
@@ -50,9 +55,18 @@ def process_referral(request, code):
     referral.respond(request, "RESPONDED")
     max_age = getattr(settings, "PINAX_COOKIE_MAX_AGE", None)
     try:
-        response = redirect(request.GET[
-            getattr(settings, "PINAX_REFERRALS_REDIRECT_ATTRIBUTE", "redirect_to")]
+        next_url = request.GET[
+            getattr(settings, "PINAX_REFERRALS_REDIRECT_ATTRIBUTE", "redirect_to")
+        ]
+        url_is_safe = url_has_allowed_host_and_scheme(
+            url=next_url,
+            allowed_hosts=request.get_host(),
```

Comment:
The code looks good :+1:  I wasn't aware that this was allowed.

I don't know if there are valid use cases where the domain differs, but I think the redirect_to domain restriction deserves a mention in the README and/or the changelog, because it could be potentially not backwards compatible for some people.